bases/simple-app/@{{ cookiecutter.project_slug }}/deploy/infra/variables.tf

```diff
@@ -18,5 +29,16 @@ variable "github_repo" {
 variable "github_org" {
   description = "Git organization name"
   type        = string
-  default     = "dnd-it"
+  default     = "@{{ cookiecutter.github_org }}"
+}
+
+variable "acm_domain_names" {
+  description = "Main domain name"
+  type        = string
+  default     = ""
```

Comment:
I will remove the default. Not sure what will happen to create an ACM certificate with "" ?